README.md

```diff
@@ -18,10 +18,18 @@ The `main()` function outputs on the serial interface information about the hard
 1. Connect a USB cable between the USB port on the target and the host computer.
 1. Run the following command to build the example project and program the microcontroller flash memory:
 
-   ```bash
-   $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm
-   ```
-   
+    * Mbed CLI 2
```

Comment:
Please also update the first section above regarding Mbed CLI. We could use https://github.com/ARMmbed/mbed-os-example-blinky#mbed-os-build-tools as a template.